docs/docs/reference/contextual/derivation-macro.md

```diff
@@ -169,14 +169,14 @@ object Eq {
       def eqv(x: T, y: T): Boolean = body(x, y)
     }
 
-  def summonAll[T: Type](using qctx: Quotes): List[Expr[Eq[_]]] = Type.of[T] match {
+  def summonAll[T: Type](using Quotes): List[Expr[Eq[_]]] = Type.of[T] match {
     case '[String *: tpes] => '{ summon[Eq[String]] }  :: summonAll[tpes]
     case '[Int *: tpes]    => '{ summon[Eq[Int]] }     :: summonAll[tpes]
     case '[tpe *: tpes]   => derived[tpe] :: summonAll[tpes]
     case '[EmptyTuple] => Nil
   }
 
-  given derived[T: Type](using qctx: Quotes) as Expr[Eq[T]] = {
+  given derived[T: Type](using q: Quotes) as Expr[Eq[T]] = {
     import qctx.reflect._
```

Comment:
If we use `q`, then it's more consistent to have `import q.reflect._`. The same for other places.

Reply:
That one is waiting for #10442. Same in other places.